upi/openstack/update_master_ignition.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh
```

Comment:
Why one shell file (update_master_ignition.sh) and one python file (update_bootstrap_ign.py)?